ui/raidboss/data/06-ew/raid/p4n.ts

```diff
@@ -142,36 +140,34 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     // Call outs can be adjusted to either merge all to one or give them directional knockback call out
-    // maybe adjust delay timer
     {
       id: 'P4N Northerly Shift Knockback',
       type: 'StartsUsing',
       netRegex: NetRegexes.startsUsing({ id: '6DAE', source: 'Hesperos', capture: false }),
-      delaySeconds: 2,
+      delaySeconds: 4,
       response: Responses.knockback(),
     },
     {
       id: 'P4N Easterly Shift Knockback',
       type: 'StartsUsing',
       netRegex: NetRegexes.startsUsing({ id: '6DB0', source: 'Hesperos', capture: false }),
-      delaySeconds: 2,
+      delaySeconds: 4,
```

Comment:
I think 4 is too much here.  Isn't this a 7 second cast?

Reply:
It's very long, it is timed so that you can use anti knockback as soon as it calls.

Reply:
Sorry, I'm still not sure I understand.  This is a 7 second cast, so `delaySeconds: 2` seems more reasonable to me.  `20|2021-12-21T14:33:15.5050000-08:00|4000EA68|Hesperos|6DB0|Easterly Shift|4000EA68|Hesperos|6.70|99.99|99.99|0.00|2.79|99d974b8e242638b`

Is the knockback not from this ability?

Reply:
I'll do another round and check again to be sure :)

Reply:
Ok we went in and tested this [extensively](https://i.imgur.com/r14SKhi.png) for you and you are right 2 seconds seems to be reasonable. Fixed in a9bff4fde06ab5fc152a8cde0258437a0402bd43